adhoc_reports/parseData.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pandas as pd
```

Comment:
Whew, pandas is a pretty large library. Never a good idea to import the entire pandas just to use one or two methods. Import what you need.